.rubocop.yml

```diff
@@ -7,16 +7,36 @@ Style/Documentation:
 Style/RescueModifier:
   Enabled: false
 
+Layout/LineLength:
+  Exclude:
+    - spec/**/*_spec.rb
+
 Metrics/BlockLength:
   Exclude:
     - spec/**/*_spec.rb
 
+Metrics/ClassLength:
+  CountAsOne:
```

Comment:
copying some common styles I use, I tend to find the default strictness to be a bit of a headache